app-misc/calamares-gentoo-livecd/calamares-gentoo-livecd-3.3.14_p20250915-r1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="8"
+
+MY_PN="calamares"
+MY_P="${MY_PN}-${PV}"
+PYTHON_COMPAT=( python3_{11..13} )
+
+inherit desktop python-single-r1
+
+DESCRIPTION="Gentoo Linux ${MY_PN} installer configuration for LiveCD"
+HOMEPAGE="https://github.com/StefanCristian/calamares-gentoo-livecd"
+SRC_URI="https://github.com/StefanCristian/${PN}/archive/refs/tags/${PV}.tar.gz -> ${PF}.tar.gz
+		mirror://gentoo/gentoo-artwork-livecd-2007.0.tar.bz2
+		mirror://gentoo/gentoo-artwork-0.2.tar.bz2"
+
+LICENSE="GPL-2"
```

Comment:
Also there is no license file in the repo, please ask upstream to add one :)  You should prefer `GPL-2+` or `GPL-3`, since calamares itself is `GPL-3`.